mirgecom/boundary.py

```diff
@@ -201,4 +200,4 @@ def exterior_grad_q(self, discr, grad_q, btag, **kwargs):
         for ispec in range(num_species):
             result[dim+2+ispec] = -int_cv.species_mass[ispec]
 
-        return join_conserved_vectors(dim, result)
+        return np.stack(result, axis=0)
```

Comment:
This one might not be needed, since `result` is already the shape of `grad_q`. I left it out in my testing and it seemed to work.

Reply:
I wondered about that. It might get better still. Right now it kind of does what `join_conserved_vectors` used to do to get the proper form.  Now maybe we can use `join_conserved` directly.

Reply:
See how this grabs ya. (c539a14)